docker/r10k/Dockerfile

```diff
@@ -1,3 +1,11 @@
+FROM alpine:3.9 as build
+
+RUN apk add --no-cache ruby git
+RUN mkdir /workspace
+WORKDIR /workspace
+COPY . /workspace
```

Comment:
I think `cwd` in Azure is not quite the same, so `.` might be a different set of files? Could verify with

```Dockerfile
RUN ls -l /workspace
```

Reply:
I think this is due to a change in the shared build.ps1, testing